crates/gpui/src/window.rs

```diff
@@ -783,6 +783,8 @@ impl Window {
             platform_window.set_app_id(&app_id);
         }
 
+        platform_window.map_window().unwrap();
```

Comment:
Do we need to add a whole platform window API here, or can we do this implicitly as part of `set_app_id`?